module/Photo/src/Photo/Service/Metadata.php

```diff
@@ -66,19 +66,20 @@ private function frac2dec($str)
 
     /**
      * Computes the shutter speed from the exif data.
-     * @param array $exif the exif data extracted from the photo.
+     *
+     * @param string $shutterSpeed the shutter speed as listed in the photo's exif data.
      *
      * @return string the shutter speed, represented as a rational string.
      */
-    private function exifGetShutter($exif)
+    private function exifGetShutter($shutterSpeed)
     {
-        if (!isset($exif['ShutterSpeedValue'])) {
-            return "unknown";
+        if (!isset($shutterSpeed)) {
+            return null;
         }
```

Comment:
Inderdaad, maar er kan een waarde meegegeven worden die niet geset is. Ik zal deze check verplaatsen naar buiten deze functie. Zodat deze functie niet wordt aangeroepen als die waarde leeg is.